INTEGRATION [PR#441 > development/1.1] This is the initial commit for the Reference Guide source

bert-e opened the pull request:
This pull request has been created automatically.
It is linked to its parent pull request #441.

**Do not edit this pull request directly.**
If you need to amend/cancel the changeset on branch
`w/1.1/feature/ZENKO-558/Ref_Guide_initial_merge`, please follow this
procedure:

```bash
 $ git fetch
 $ git checkout w/1.1/feature/ZENKO-558/Ref_Guide_initial_merge
 $ # <amend or cancel the changeset by _adding_ new commits>
 $ git push origin w/1.1/feature/ZENKO-558/Ref_Guide_initial_merge
```

Please always comment pull request #441 instead of this one.

wabernat committed:
feature/ZENKO-558/Ref_Guide_initial_merge

This is the initial commit for the Reference Guide source

Please do not review in depth. Say yes. Then file tickets.

wabernat committed:
This is the initial commit for the Reference Guide source

Please do not review in depth. Say yes. Then file tickets.

wabernat committed:
feature/ZENKO-558/Ref_Guide_initial_merge

feature/ZENKO-558/Ref_Guide_initial_merge

feature/ZENKO-558/Ref_Guide_initial_merge

This is the initial commit for the Reference Guide source

Please do not review in depth. Say yes. Then file tickets.

wabernat committed:
feature/ZENKO-558/Ref_Guide_initial_merge

This is the initial commit for the Reference Guide source

Please do not review in depth. Say yes. Then file tickets.

bert-e committed:
Merge branch 'feature/ZENKO-558/Ref_Guide_initial_merge' into tmp/octopus/w/1.1/feature/ZENKO-558/Ref_Guide_initial_merge

ghost commented:
There were the following issues with this Pull Request

* Commit: 5197a5d703fec92b41829682863f83d9fa9f166e
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues

wabernat committed:
Merge branch 'feature/ZENKO-558/Ref_Guide_initial_merge' of https://github.com/scality/Zenko into feature/ZENKO-558/Ref_Guide_initial_merge

bert-e committed:
Merge branch 'feature/ZENKO-558/Ref_Guide_initial_merge' into tmp/octopus/w/1.1/feature/ZENKO-558/Ref_Guide_initial_merge

ghost commented:
There were the following issues with this Pull Request

* Commit: 5197a5d703fec92b41829682863f83d9fa9f166e
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: 994625b9ce9f9ed4146700c9eca4ecb781fb3a53
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues

wabernat committed:
Merge branch 'feature/ZENKO-558/Ref_Guide_initial_merge' of https://github.com/scality/Zenko into feature/ZENKO-558/Ref_Guide_initial_merge

bert-e committed:
Merge branch 'feature/ZENKO-558/Ref_Guide_initial_merge' into tmp/octopus/w/1.1/feature/ZENKO-558/Ref_Guide_initial_merge

ghost commented:
There were the following issues with this Pull Request

* Commit: 5197a5d703fec92b41829682863f83d9fa9f166e
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: 994625b9ce9f9ed4146700c9eca4ecb781fb3a53
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: 136b3ac362e9eb375fbac01f82c36ca4f5369b8a
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues

wabernat committed:
Feature/ZENKO-558/Ref_Guide_initial_merge

Merge branch 'feature/ZENKO-558/Ref_Guide_initial_merge' of https://github.com/scality/Zenko into feature/ZENKO-558/Ref_Guide_initial_merge

bert-e committed:
Merge branch 'feature/ZENKO-558/Ref_Guide_initial_merge' into tmp/octopus/w/1.1/feature/ZENKO-558/Ref_Guide_initial_merge

ghost commented:
There were the following issues with this Pull Request

* Commit: 5197a5d703fec92b41829682863f83d9fa9f166e
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: 994625b9ce9f9ed4146700c9eca4ecb781fb3a53
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: 136b3ac362e9eb375fbac01f82c36ca4f5369b8a
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: 78f38fde0eee08d9845daee0b78740199526a687
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues
